src/main/java/org/zalando/nakadi/config/ValidatorConfig.java

```diff
@@ -39,7 +38,6 @@ public SchemaEvolutionService schemaEvolutionService() throws IOException {
 
         final List<SchemaEvolutionConstraint> schemaEvolutionConstraints = Lists.newArrayList(
                 new CompatibilityModeChangeConstraint(),
-                new CompatibleSchemaChangeConstraint(),
```

Comment:
👍 my mistake. The base branch I sent to @adyach didn't contain this change, which is necessary to test this feature.